tools/flaky-test-retryer/github_commenter.go

```diff
@@ -158,19 +170,28 @@ func buildNewComment(jd *JobData, entries map[string]int, outliers []string) str
 	}
 	sort.Strings(keys)
 	for _, test := range keys {
-		entryString = append(entryString, fmt.Sprintf("%s | %d/%d", test, entries[test], maxRetries))
+		entryString = append(entryString, fmt.Sprintf("%s | %s | %d/%d", test, buildLinks(entries[test].oldLinks, jd.GCSPath), entries[test].retries, maxRetries))
 	}
 	return fmt.Sprintf(commentTemplate, identifier, strings.Join(entryString, "\n"), cmd)
 }
 
+// buildLinks generates Markdown-formatted Spyglass links from GCS paths
+func buildLinks(oldLinks string, GCSPath string) string {
+	formattedLink := fmt.Sprintf("[link](%s)", strings.Replace(GCSPath, gcsPrefix, spyglassPrefix, -1))
```

Comment:
would prefer something stronger than replace all

Reply:
ReportMessage's `URL` field turns out to be the link we need already